docs/library/bluetooth.rst

```diff
@@ -514,19 +512,24 @@ writes from a client to a given characteristic, use
 
     Sends a notification request to a connected client.
 
-    If *data* is not ``None``, then that value is sent to the client as part of
-    the notification. The local value will not be modified.
-
     Otherwise, if *data* is ``None``, then the current local value (as
     set with :meth:`gatts_write <BLE.gatts_write>`) will be sent.
 
+    If *data* is not ``None``, then that value is sent to the client as part of
+    the notification. The local value will not be modified.
```

Comment:
It looks like this paragraph was inadvertently moved here?  Because it no longer reads properly.

Reply:
Yep. I've kept the order because i think it's clearer to go default first then override second, but fixed the wording.